lib/CMakeLists.txt

```diff
@@ -48,4 +48,15 @@ target_include_directories(disccord PUBLIC ../include)
 target_link_libraries(disccord ${LIBS})
 
 install(TARGETS disccord DESTINATION /usr/local/lib)
-install(DIRECTORY ../include DESTINATION /usr/local/include/disccord)
+file(GLOB HEADERS_DISCCORD ../include/disccord/*.hpp)
```

Comment:
yeah good do it that way too.except ill be doing `${CMAKE_SOURCE_DIR}/include/disccord` otherwise all the files will install to `usr/local/include/include/disccord/stuff` instead of `usr/local/include/disccord/stuff`